library/src/main/kotlin/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -311,4 +316,17 @@ internal class TransactionPayloadFragment :
                 }
             }
     }
+
+    private fun CharSequence.lines(): List<CharSequence> {
+        val linesList = this.lineSequence().toList()
+        val result = mutableListOf<CharSequence>()
+        var lineIndex = 0
+        for (index in linesList.indices) {
+            result.add(subSequence(lineIndex, lineIndex + linesList[index].length))
+            lineIndex += linesList[index].length + 1
+        }
+        if (result.isEmpty())
+            result.add(subSequence(0, length))
```

Comment:
Why this?

Reply:
That's because when the char sequence doesn't have any lines so I should add all of its content to the result.